content/blog/wingingit/winging-it-01.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 1. CSS Cascade Layers
+author:
+  - miriam
+  - ed
+  - stacy
+date: 2023-07-09
+tags:
+  - CSS
+  - Cascade Layers
+length: 40 mins
+action:
+  text: Watch on YouTube »
+  url: https://youtu.be/FwICaSE8iuY
+image:
+  src: winging-it/winging-it-1.jpg
```

Comment:
Might be nice to have a bit more uniqueness in these, specific to the episode? But it looks really great.

Reply:
It's very not-urgent. It's a great idea to have a template we can use without a lot of customization - might just be good (eventually) to have a way to slot in a screen-grab of a demo or something else from the video. Fine if we want to ship with something simpler, tho.